### DISCUSSION TITLE
Same augmentations for frame sequence

### DISCUSSION THREAD

**Discussion by gorodion** [albumentations-team/albumentations#1177] [Ideas]
Let's say we solve the problem of classification by sequence of frames (as action recognition). I would like to be able to apply the same augmentation for N frames. I think it's worth adding the "images" parameter, which contains a list of frames (similar to "masks")

**Comment by Dipet** [comment]
You could try to use [additional targets](https://albumentations.ai/docs/examples/example_multi_target/) or [ReplayCompose](https://albumentations.ai/docs/examples/replay/) for this
